src/main/graphql/GetConsignment.graphql

```diff
@@ -2,5 +2,11 @@ query getConsignment($consignmentId: UUID!) {
     getConsignment(consignmentid: $consignmentId) {
         userid
         seriesid
+        totalFiles
+        fileCheckProgress {
```

Comment:
The names for these will need to be updated based on this change: https://github.com/nationalarchives/tdr-consignment-api/pull/57/files#r453722445